v3/setup/capcom.sh

```diff
@@ -7,26 +7,24 @@ if [ "${NODE_ROLE}" != "control" ]  && [ "${NODE_ROLE}" != "it-hybrid" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get capcom-bootstrapped)" == "true" ]; then
-    exit 0
+etcdctl get /capcom/config/proxy
```

Comment:
hm ok. Then a parent namespace it is.
What about:
```
etcdctl set /bootstrap/capcom true
```

The semantics and namespace implies that it belongs to *some* kind of bootstrap process. If we ever want to use the `/bootstrap/capcom` key for anything else, we can just extend it.